src/Http/Http.Extensions/gen/DiagnosticDescriptors.cs

```diff
@@ -26,14 +26,12 @@ internal static class DiagnosticDescriptors
     );
 
     // This is temporary. The plan is to be able to resolve all parameters to a known EndpointParameterSource.
-    public static DiagnosticDescriptor GetUnableToResolveParameterDescriptor(string parameterName)
-    {
-        return new(
-            "RDG073",
+    // For now, we emit a warning for the unsupported set.
+    public static DiagnosticDescriptor UnableToResolveParameterDescriptor { get;  } = new(
+            "RDG003",
```

Comment:
Do we have these diagnostic IDs documented anywhere? In dotnet/runtime we have https://github.com/dotnet/runtime/blob/main/docs/project/list-of-diagnostics.md so we ensure we don't overlap/reuse IDs.

Reply:
We don't...our typically progress is to document the diagnostic IDs in our docs [https://aka.ms/aspnet/analyzers] as soon as possible. Most of our diagnostic IDs come out of analyzers though.

I assume the doc you linked is manually updated?

Reply:
Correct. That doc is manually updated as we add the diagnostic in the repo. The docs part comes later.

Reply:
I filed https://github.com/dotnet/aspnetcore/issues/47287. I notice the rutnime repo does a good job of keeping all their diagnostics codes under the same prefix. That might be a practice we should adopt here. If we do, it would make the in-repo doc necessary as a way for different sources to earmark the diagnostic code ranges they intend on using...